src/components/GenerateTransfersMenu.tsx

```diff
@@ -35,10 +35,15 @@ export const GenerateTransfersMenu = (props: GenerateTransfersMenuProps): JSX.El
     let drainCSV = "token_type,token_address,receiver,value,id,";
     if (drainAddress) {
       assetBalance?.forEach((asset) => {
+        console.log(assetBalance);
         if (asset.token === null && asset.tokenAddress === null) {
           const decimalBalance = fromWei(new BigNumber(asset.balance), 18);
+          console.log(asset.balance);
+
+          console.log(decimalBalance.toFixed());
```

Comment:
I think these logs should either be removed or have some descriptive strings attached to them.